ext/date/php_date.c

```diff
@@ -1008,7 +1008,7 @@ PHPAPI zend_long php_parse_date(const char *string, zend_long *now)
 PHP_FUNCTION(strtotime)
 {
 	zend_string *times;
-	int error1, error2;
+	int error1, epoch_does_not_fit_in_zend_long;
```

Comment:
is there is only one error now, I would suggest to change `error1` to `error`, or perhaps to the more descriptive `parse_error`.